sdk/identity/identity/review/identity.api.md

```diff
@@ -106,26 +117,33 @@ export interface IdentityClientOptions extends ServiceClientOptions {
 
 // @public
 export class InteractiveBrowserCredential implements TokenCredential {
-    constructor(tenantId: string, clientId: string, options?: InteractiveBrowserCredentialOptions);
+    constructor(options?: InteractiveBrowserCredentialOptions);
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
 }
 
 // @public
 export interface InteractiveBrowserCredentialOptions extends IdentityClientOptions {
+    clientId?: string;
     loginStyle?: BrowserLoginStyle;
     postLogoutRedirectUri?: string | (() => string);
     redirectUri?: string | (() => string);
+    tenantId?: string;
 }
 
 // @public
 export const logger: import("@azure/logger").AzureLogger;
 
 // @public
 export class ManagedIdentityCredential implements TokenCredential {
-    constructor(clientId?: string, options?: IdentityClientOptions);
+    constructor(options?: ManagedIdentityCredentialOptions);
```

Comment:
This should have an overload with clientId rather than moving the clientId to the options bag. The clientId is used to specify a user assigned identity and adding the overload gives us a chance to document this.

Reply:
Done